addons/medical_feedback/XEH_postInit.sqf

```diff
@@ -80,10 +88,28 @@ GVAR(heartBeatEffectRunning) = false;
         _new setVariable ["acre_sys_core_isDisabled", _status, true];
     };
     [QUOTE(ADDON), VOL_UNCONSCIOUS, _status] call EFUNC(common,setHearingCapability);
-    [_status, 0] call FUNC(effectUnconscious);
+    [true] call FUNC(handleEffects);
     ["unconscious", _status] call EFUNC(common,setDisableUserInputStatus);
 }] call CBA_fnc_addPlayerEventHandler;
 
+// Forced say3D
+[QGVAR(forceSay3D), {
+    params ["_unit", "_sound", "_distance"];
+
+    if (ACE_player distance _unit > _distance) exitWith {};
+
+    if (vehicle _unit == _unit) then {
+        // say3D waits for the previous sound to finish, so use a dummy instead
```

Comment:
would normal `say` work ok here? we don't really need 3d positioning

Reply:
No, because say waits for a previous say.